Source/WebCore/crypto/keys/CryptoKeyOKP.cpp

```diff
@@ -216,6 +213,11 @@ std::optional<CryptoKeyPair> CryptoKeyOKP::platformGeneratePair(CryptoAlgorithmI
     return { };
 }
 
+bool CryptoKeyOKP::platformCheckPairedKeys(CryptoAlgorithmIdentifier, NamedCurve, Vector<uint8_t>&&, Vector<uint8_t>&&)
```

Comment:
Should they be const& then?

Reply:
yeah, will submit a new patch fixing this. Thanks

Reply:
Done